catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSource.java

```diff
@@ -298,6 +302,7 @@ public void refresh(Map<String, Object> configuration) throws SecurityServiceExc
     String usernameValue = (String) configuration.get(USERNAME_KEY);
     Boolean disableCnCheckProp = (Boolean) configuration.get(DISABLE_CN_CHECK_KEY);
     String id = (String) configuration.get(ID_KEY);
+    Boolean allowRedirects = (Boolean) configuration.get(ALLOW_REDIRECTS_KEY);
```

Comment:
Should we check for null returns here?

Reply:
this comes from the configuration map passed by blueprint; i don't think we've been checking it in other sources either